src/server/api/cycleData/index.ts

```diff
@@ -152,6 +153,7 @@ export const CycleDataApi = {
       AuthMiddleware.requireEditAssessmentFile,
       createFile
     )
+    express.get(ApiEndPoint.CycleData.Repository.file(), AuthMiddleware.requireViewFile, getRepositoryFile)
```

Comment:
This is where we diverge a bit from previous implementation, we add the authorizer here because it's very fast to query the `cycleSchema.repository` table, so we can check the rights in the authorizer

The authorizer itself is 1:1, using the new RepositoryItem

The old countryFile authorizer has been deprecated and will be removed in later PRs